Gulpfile.ts

```diff
@@ -215,8 +215,8 @@ for (const i in libraryTargets) {
             .pipe(gulp.dest(".")));
 }
 
-const configureNightlyJs = path.join(scriptsDirectory, "configureNightly.js");
-const configureNightlyTs = path.join(scriptsDirectory, "configureNightly.ts");
+const configureNightlyJs = path.join(scriptsDirectory, "configurePrerelease.js");

```

Comment:
nit. maybe we should change the name of the variable as well to configurePrerelease